### PR TITLE
Twk

### PR DESCRIPTION
**Pull request by ntejos**
Added an extra parameter for writelinepars, so now it gets the spectrum filename directly from the spectrum  that is loaded (instead from cfg.filename).

### CODE REVIEW

File: joebvp/cfg.py [hunk 1]
```diff
@@ -17,13 +17,17 @@
...
+lsf='COS_LP3'
+instr=['COS','COS']
+gratings=['G130M','G160M']
+
+slits=['NA','NA']
+lsfranges=np.array([[1100,1400],[1300,1800]])
+lps=['3','3']
+cen_wave=['1327','1600']
+
 echarge=4.803204505713468e-10
```
Comment (jnburchett) [inline joebvp/cfg.py]: Would you mind putting that extra instrument/grating info back into the cfg.py file and recommit?  Todd uses all those settings and I'd like to keep them in for now.  Probably need to come up with a better solution for the long term....

**Review by jnburchett — CHANGES_REQUESTED** [review]
I love it.  Thanks!  One annoying request.

File: joebvp/cfg.py (hunk 1 above)
Comment (ntejos) [reply joebvp/cfg.py]: Sure, no problem, I came up with a patch for now but I agree it would be nice to come up with a better long-term solution.

**Review by jnburchett — APPROVED** [review]
Nice!  Todd will be happy to know that he has graduated beyond 'casual user'.

**Pull request merged**